### DISCUSSION TITLE
Greater Than node but in Malt format?

### DISCUSSION THREAD

**Discussion by Psi25** [bnpr/Malt#316] [Q&A] [answered]
I've been following this tutorial(https://www.youtube.com/watch?v=VcaRAhif9ec) that used the Greater Than node, and I'm having problem replicating it on Malt.

**Comment by pragma37** [comment, marked as answer]
You can search nodes by name using the [Operator Search](https://docs.blender.org/manual/en/2.82/interface/controls/templates/operator_search.html) menu.
Just search for *"greater than"*.

**Reply by Psi25** [reply]
Thanks. I had to use "float greater" since there's no vec3 or vec4. I had to split the color channels into 3. 

The node setup is like this. It's meant to replicate cel-shading for gradient textures. the value is either 1(white) or 0(black). 

![2022-06-07 10 34 01](https://user-images.githubusercontent.com/103315144/172283501-b0134b00-f693-4e45-994d-cd91deee0cab.png)

To get a non cel-shaded look: wire the Texture's color channel to "if true" of the "float if else" node.

**Reply by pragma37** [reply]
I see, you can use the *rgba gradient* node for that as well:
![imagen](https://user-images.githubusercontent.com/36610999/172393945-1b77a5ab-33ea-4d43-8f39-d4bffaf2ab94.png)
